src/wallet/abstract-hd-electrum-wallet.js

```diff
@@ -68,7 +68,7 @@ export class AbstractHDElectrumWallet extends AbstractHDWallet {
   }
 
   async generate() {
```

Comment:
@xsats if you feel strongly you need 256 bit for your app, how about we pass in a flag here:

```js
generate({ highEntropy: true })
```

Reply:
Sounds like a good practical solution to me